src/components/theme/Image/Image.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { /* useState, useEffect, useCallback, */ useRef } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { getImageAttributes } from '@plone/volto/components/theme/Image/utils';
+
+/**
+ * Image component
+ * @param {object | string} image - Plone image as object or url
+ * @param {string} alt - Alternative text for image
+ * @param {string} className - CSS class attribute
+ * @param {string} pictureClassName - CSS class attribute for picture element
+ * @param {string} size - (default: full) Size of image: can be small, medium, large, full
+ * @param {string} responsive - if the image is responsive it adapts to the container
+ * @param {string} role - img role attribute
+ * @param {boolean} loading - (default: eager) if set to `lazy`, the srcset is loaded when the image is in the viewport
+ */
+const Image = ({
+  image,
+  alt = '',
+  className,
+  pictureClassName,
+  size = 'full',
+  responsive = true,
```

Comment:
I'm not sure if the correct name for this is `responsive`, like frameworks (e.g. Bootstrap) usually say, or maybe `fluid`, which is the actual behavior that this gives to the image. Default to true because it is more backwards compatible and is the more common option imho.